### DISCUSSION TITLE
Ollama error 404

### DISCUSSION THREAD

**Discussion by AlessioGesuelliUnicam** [langflow-ai/langflow#1710] [Q&A]
I don't know how resolve this problem
Ollama is running but langflow return Error 404

 
ValueError: Error: Ollama call                     
                             failed with status code 404. Maybe                 
                             your model is not found and you                    
                             should pull the model with `ollama                 
                             pull llama2:latest`.            
 - ERROR    - chat -  chat.py:57
                             Error in chat websocket: control frame             
                             too long                                           

![Acquisizione schermata 16 04 2024 alle 16 32 37  1](https://github.com/langflow-ai/langflow/assets/105433120/8863560f-3bfd-408e-820c-d661bbe216ee)
![Acquisizione schermata 16 04 2024 alle 16 32 37  2](https://github.com/langflow-ai/langflow/assets/105433120/a824ed5a-306d-4cf9-9b0e-6173a76a11f5)
![Acquisizione schermata 16 04 2024 alle 16 32 57  1](https://github.com/langflow-ai/langflow/assets/105433120/9cee97b6-dd27-496c-9a31-6680e5670198)

**Comment by Cristhianzl** [comment]
Hi! 👋

We are using other channels as our official means of communication with users. We apologize for the delayed response.
For faster assistance, please direct your questions to our Discord channel or open a GitHub issue, as we are much more active on these platforms.

[Langflow Discord Server](https://discord.com/invite/EqksyE2EX9)

Thank you for your understanding.

Best regards,
The Langflow Team
